### DISCUSSION TITLE
source zmq not found

### DISCUSSION THREAD

**Discussion by iceteas** [lf-edge/ekuiper#1699] [General]
I am trying to add zmq sources for ekuiper, Compilation completed and source configuration added. The service was restarted, but an error was reported during the query
![image](https://user-images.githubusercontent.com/12604419/227930526-b2aa6cf1-a762-4394-aae7-7155f67907f9.png)

**Comment by iceteas** [comment]
![image](https://user-images.githubusercontent.com/12604419/227932834-6e323d1a-0e3c-481b-a388-47300d2a9c7d.png)

**Comment by iceteas** [comment]
install libzmq3-dev through apt-get install, no errors will be reported

**Reply by ngjaying** [reply]
Thank you for updating your finding
